device/mellanox/x86_64-mlnx_msn4600c-r0/sensors.conf.a1

```diff
@@ -0,0 +1,145 @@
+##
+## Copyright (c) 2020-2022 NVIDIA CORPORATION & AFFILIATES.
```

Comment:
please change to 2022 only as this is new file

Reply:
done